src/inputs/Radio.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div>
+        <label v-for="item in selectItems" :key="item.value">
+            <input
+                type="radio"
+                :name="item.label"
+                v-on:input="e => onChange(item.value)" />
```

Comment:
this doesn't read an initial value.
(try adding a value to `App.vue`s 'formData')